infra/storage/file_transfer_managers/s3Mgr.class.php

```diff
@@ -122,9 +131,12 @@ protected function doLogin($sftp_user, $sftp_pass)
 			return false;
 		}
 
-		if($this->s3Arn && (!isset($sftp_user) || !$sftp_user) && (!isset($sftp_pass) || !$sftp_pass))
+		if($this->s3Arn && $this->s3Region && (!isset($sftp_user) || !$sftp_user) && (!isset($sftp_pass) || !$sftp_pass))
 		{
-			KalturaLog::debug('Found env VAR from config- ' . $this->s3Arn);
+			$s3Arn = $this->s3Arn;
```

Comment:
Why do you define these local params? if its only for the log simply use the values stored on $this